frontend/public/components/import-yaml.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { CreateYAML } from './create-yaml';
+
+export class ImportYamlPage extends React.Component {
```

Comment:
Make this a stateless functional component.

https://medium.com/@ethan_ikt/react-stateless-functional-component-with-typescript-ce5043466011